lms/static/js/dashboard/views/course_view.js

```diff
@@ -0,0 +1,161 @@
+                        courseNumber = $currentTarget.data("course-number"),
+                        courseName = $currentTarget.data("course-name"),
+                        certNameLang = $currentTarget.data("cert-name-long"),
+                        refundInfo = $currentTarget.data("refund-info");
+
+                    $('#track-info').html(_.sprintf(track_info, {
+                        course_number: _.sprintf(
+                            "<span id='unenroll_course_number'>%(courseNumber)s</span>", {courseNumber: courseNumber}
+                        ),
+                        course_name: _.sprintf(
+                            "<span id='unenroll_course_name'>%(courseName)s</span>", {courseName: courseName}
+                        ),
+                        cert_name_long: _.sprintf(
+                            "<span id='unenroll_cert_name'>%(certNameLang)s</span>", {certNameLang: certNameLang}
+                        )
+                    }, true));
+
+                    $('#refund-info').html(refundInfo);
+                    $("#unenroll_course_id").val(courseId);
+                },
+
+                emailSettings: function (e) {
+                    var $currentTarget = this.$(e.currentTarget);
+
+                    $("#email_settings_course_id").val($currentTarget.data("course-id"));
+                    $("#email_settings_course_number").text($currentTarget.data("course-number"));
+
+                    if ($currentTarget.data("optout") === "False") {
+                        $("#receive_emails").prop('checked', true);
+                    }
+                },
+
+                upgradeToVerified: function (e) {
+                    var $currentTarget = this.$(e.currentTarget);
+                    $currentTarget.closest(".action-upgrade").data("user");
+                    $currentTarget.closest(".action-upgrade").data("course-id");
+                },
+
+                unRegisterBlockCourse: function (e) {
+                    var $currentTarget = this.$(e.currentTarget),
+                        courseId = $currentTarget.data("course-id"),
+                        courseNumber = $currentTarget.data("course-number"),
+                        courseName = $currentTarget.data("course-name");
+
+                    if (this.$('#block-course-msg').length) {
+                        $('.disable-look-unregister').click();
+                    }
+
+                    $('#track-info').html(_.sprintf(
+                        "<span id='unenroll_course_number'>%(courseNumber)s</span> " +
+                        "- <span id='unenroll_course_name'>%(courseName)s?</span>",
+                        {courseNumber: courseNumber, courseName: courseName})
+                    );
+
+                    $("#unenroll_course_id").val(courseId);
+                }
+            });
+
+            return CourseView;
+        });
+
+}).call(this, define || RequireJS.define, accessible_modal); // jshint undef:false
```

Comment:
Please update this file to consistently use single quotes. It could help if you ran JSHint locally which would show warnings about this sort of thing.